rust/types/src/common.rs

```diff
@@ -0,0 +1,26 @@
+use std::fmt;
+
+#[derive(Debug, Clone, Copy)]
+pub enum ChromaS3FilePrefixes {
```

Comment:
I think these should be colocal with the source crates, not centralized. 

Also, SparseIndex prefix exists